### ISSUE TITLE
bug: ReVanced Manager is not able to compile, apparently

### ISSUE THREAD

**Issue by joseph-20** [revanced/revanced-manager#68]
### Type

Error while running the manager

### Branch

Flutter

### Bug description

ReVanced Manager crashes in the middle of patching the apk

### Steps to reproduce

1. Download and install a compatible YouTube apk
2. Choose patches
3. Start patching
4. The app will crash while still patching the apk

### Relevant log output

```shell
FATAL EXCEPTION: Thread-9
Process: app.revanced.manager.flutter, PID: 20761
brut.androlib.AndrolibException: brut.common.BrutException: could not exec (exit code = 139): [/data/app/~~YSrPDIPLQKA6jJIvGGtf6w==/app.revanced.manager.flutter-i6rQfS1M5kBDTXroPmMiOA==/lib/arm64/libapp.so, compile, --dir, /data/user/0/app.revanced.manager.flutter/cache/patcher/tmp-IIVIFI/cache/res, --legacy, -o, /data/user/0/app.revanced.manager.flutter/cache/patcher/tmp-IIVIFI/cache/build/resources.zip]
 at brut.androlib.res.AndrolibResources.aapt2Package(AndrolibResources.java:372)
 at brut.androlib.res.AndrolibResources.aaptPackage(AndrolibResources.java:626)
 at app.revanced.patcher.Patcher.save(Patcher.kt:212)
 at app.revanced.manager.flutter.MainActivity.runPatcher$lambda-18(MainActivity.kt:286)
 at app.revanced.manager.flutter.MainActivity.$r8$lambda$-gNqYPzZb-BzxULVb2BYsca-Uls(Unknown Source:0)
 at app.revanced.manager.flutter.MainActivity$$ExternalSyntheticLambda12.run(Unknown Source:24)
 at java.lang.Thread.run(Thread.java:920)
Caused by: brut.common.BrutException: could not exec (exit code = 139): [/data/app/~~YSrPDIPLQKA6jJIvGGtf6w==/app.revanced.manager.flutter-i6rQfS1M5kBDTXroPmMiOA==/lib/arm64/libapp.so, compile, --dir, /data/user/0/app.revanced.manager.flutter/cache/patcher/tmp-IIVIFI/cache/res, --legacy, -o, /data/user/0/app.revanced.manager.flutter/cache/patcher/tmp-IIVIFI/cache/build/resources.zip]
 at brut.util.OS.exec(OS.java:99)
 at brut.androlib.res.AndrolibResources.aapt2Package(AndrolibResources.java:368)
 ... 6 more
```

### Additional context

Version: 0.0.1
Model: ONEPLUS A6003
Android Version: 12 (It's actually Android 12L, SDK level 32)
Arch: arm64-v8a

**Comment by Aunali321** [comment]
Did you give storage permissions?

**Comment by Sculas** [comment]
This is a known issue, a fix will be pushed soon. Sorry for the inconvenience!

**Comment by DcR-NL** [comment]
Same issue here when patching v17.33.42. For the storage permission, I enabled "Allow access to media only", since the other only option is "Don't allow".

Version: 0.0.1
Model: ONEPLUS A6003
Android Version: 11
Arch: arm64-v8a

**Comment by joseph-20** [comment]
@Sculas nice to hear that, no worries and thank a lot!

**Comment by Sculas** [comment]
This was fixed in 6e4837da27ea2ce0895939147df91405510a24fa.
